packages/block-editor/src/components/block-variation-transforms/README.md

```diff
@@ -0,0 +1,67 @@
+# Block Variation Transforms
+
+This component allows to display the selected block's variations which have the `transform` option set in `scope` property and to choose one of them.
+
+By selecting such a variation an update to the selected block's attributes happen, based on the variation's attributes.
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Renders the selected block's variations which have the `transform` option set in `scope` property.
+
+```jsx
+import { useSelect } from '@wordpress/data';
+import {
+	__experimentalBlockVariationTransforms as BlockVariationTransforms,
+} from '@wordpress/block-editor';
+
+const MyBlockVariationTransforms = () => {
+	const { selectedBlockName, selectedBlockClientId } = useSelect(
+		( select ) => {
+			const { getSelectedBlockClientId, getBlockName } = select(
+				'core/block-editor'
+			);
+			const _selectedBlockClientId = getSelectedBlockClientId();
+			const _selectedBlockName =
+				_selectedBlockClientId &&
+				getBlockName( _selectedBlockClientId );
+			return {
+				selectedBlockName: _selectedBlockName,
+				selectedBlockClientId: _selectedBlockClientId,
+			};
+		}
+	);
+
+	return (
+		<BlockVariationTransforms
```

Comment:
This example made me think, it feels like `blockName` is redundant in the public API of `BlockVariationTransforms`. You can compute it based on `selectedBlockClientId`. It would simplify this example as you would only need to call `getSelectedBlockClientId` inside `useSelect` call.

Reply:
You're right. I'll update the code.